classes/submission/form/PKPSubmissionSubmitStep1Form.inc.php

```diff
@@ -111,9 +111,10 @@ function fetch($request) {
 	 */
 	function initData($data = array()) {
 		if (isset($this->submission)) {
+			$query = $this->getCommentsToEditor($this->submissionId);
```

Comment:
`$query` could be `null` here, I believe, so the method call below (`$query->getHeadNote()`) cannot be done, or? Maybe then to use something like `$query ? $query->getHeadNote() : ''` below?